eng/tools/generator/config/validate/remoteValidator.go

```diff
@@ -55,13 +59,18 @@ func (v *remoteValidator) Validate(cfg config.Config) error {
 		}).ToSlice(&tags)
 		// check the tags one by one
 		if err := validateTagsInReadme([]byte(contentOfReadme), readme, tags...); err != nil {
+			invalidReadme[readme] = struct{}{}
 			errResult = errors.Join(errResult, err)
 		}
 		// check module-name exist
 		if err := validateModuleNameInReadmeGo([]byte(contentOfReadmeGo), readme); err != nil {
+			invalidReadme[readme] = struct{}{}
 			errResult = errors.Join(errResult, err)
 		}
 	}
+	for readme, _ := range cfg.Track2Requests {
+		delete(cfg.Track2Requests, readme)
+	}
```

Comment:
why we remove all the readme?

Reply:
my mistake, suppose to delete invalid request which collected above, use the wrong variable, should range  invalidReadme  which I renamed to invalidTrack2Requests